Directory.Build.props

```diff
@@ -114,6 +114,9 @@
   <!-- Helpful properties used elsewhere -->
   <PropertyGroup>
     <TargetFrameworkVersion>$([MSBuild]::GetTargetFrameworkVersion($(TargetFramework)))</TargetFrameworkVersion>
-    <TargetFrameworkIsNet9OrGreater Condition="$([MSBuild]::VersionGreaterThanOrEquals($(TargetFrameworkVersion), 9.0))">true</TargetFrameworkIsNet9OrGreater>
+    <IsNet8OrGreater>false</IsNet8OrGreater>
+    <IsNet8OrGreater Condition="$([MSBuild]::VersionGreaterThanOrEquals($(TargetFrameworkVersion), 8.0))">true</IsNet8OrGreater>
+    <IsNet9OrGreater>false</IsNet9OrGreater>
+    <IsNet9OrGreater Condition="$([MSBuild]::VersionGreaterThanOrEquals($(TargetFrameworkVersion), 9.0))">true</IsNet9OrGreater>
```

Comment:
question/thought:
Should we use `$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net5.0'))` instead?

I believe this would be "technically more correct".
Because extracting the `TargetFrameworkVersion` is technically ambiguous with older versions, where
e.g. `2.0` could mean
- `netcoreapp2.0`
- `netstandard2.0`
- `net20`

Well, to be fair, .NET starts with `net5.0`, which is higher than all the other Frameworks, so we should be fine, practically speaking.

But `[MSBuild]::IsTargetFrameworkCompatible` also handles _OS-specific TFMs_ correctly ... not sure if we could run with the current solution into some unexpected corner cases.

Reply:
Ew.... that doesn't sound good. 

I don't see `TargetFrameworkVersion` in the docs for SDK style projects but it looks like it exists/existed for old style projects. Easy enough to avoid - we'll just rename it to avoid conflicts.